### PR TITLE
added yarn as dependency and running locally installed yarn

### PR DESCRIPTION
**Pull request by ajaymathur**
Fixes https://github.com/boltpkg/bolt/issues/53

### CODE REVIEW

**Comment by thejameskyle** [comment]
Why peerDependency?

**Comment by ajaymathur** [comment]
yeah, it should be dependency. Updated 🙂

**Comment by lukebatchelor** [comment]
Sweet, those tests were useful to update.

File: src/utils/yarn.js [hunk 1]
```diff
@@ -3,11 +3,17 @@ import includes from 'array-includes';
...
 import type Package from '../Package';
+import Project from '../Project';
 import * as processes from './processes';
 import * as fs from '../utils/fs';
 import * as logger from '../utils/fs';
 import { DEPENDENCY_TYPE_FLAGS_MAP } from '../constants';
 
+async function getProjectBinPath(cwd: string): Promise<string> {
+  const project = await Project.init(cwd);
+  return project.pkg.nodeModulesBin;
```
Comment (thejameskyle) [inline src/utils/yarn.js]: Had to revert this in #70 because this should be pulling from the bin path of `bolt` not the project.

**Pull request merged**